app/main.py

```diff
@@ -1,3 +1,62 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __add__(self, other: int | float) -> "Distance":
```

Comment:
If your function gets only int and float types, why do you check `if isinstance(other, Distance)` in the next line and cover this case? Fix annotations everywhere